### ISSUE TITLE
Uninstalling Composer

### ISSUE THREAD

**Issue by thedirtypanda** [composer/windows-setup#39]
When you want to uninstall the composer on Windows 7, there will come a error-message, that you have to close the windows explorer (not the browser ;)) ...

**Comment by johnstevenson** [comment]
It is not actually an error message.
TLDR - it is an instruction that is part of the uninstall process

If the shell-menu feature has been installed and you right-click an item in Windows/File Explorer to bring up a context menu, then the Composer shell-menu dll gets loaded. This dll needs to be deleted when you uninstall Composer-Setup, so we ask the operating system if it is being used by any processes - otherwise the delete operation will fail. If this is the case, usually because Windows/File Explorer is still open, you are asked to close it, or any other processes,  first.

It has nothing to do with a web browser.

Is this what you meant and does this explain everything?
